CHANGELOG.md

```diff
@@ -24,6 +24,9 @@ Increment the:
 * [SDK] MeterProvider should own MeterContext, not share it
   [#2218](https://github.com/open-telemetry/opentelemetry-cpp/pull/2218)
 
+* [SDK] TracerProvider should own TracerContext, not share it
+  [#2221](https://github.com/open-telemetry/opentelemetry-cpp/pull/2221)
```

Comment:
Is this a breaking change which needs to noted out?

Reply:
Thanks, now listed as breaking in CHANGELOG